compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/text/input/internal/LegacyPlatformTextInputServiceAdapter.skiko.kt

```diff
@@ -54,7 +54,6 @@ internal actual fun legacyTextInputServiceAdapterAndService():
             }
 
             override fun stopInput() {
-                service.stopInput()
```

Comment:
Probably because I thought that it should be symmetrical and the session is a separate thing.

After looking into details your fix is indeed correct, and mine wasn't.